Dockerfile

```diff
@@ -133,8 +133,9 @@ RUN wget https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 && \
     chmod +x jq-linux64 && \
     mv jq-linux64 /usr/local/bin/jq
 
-# Add node, npm and xml-js
-RUN apt-get update && apt-get install -y nodejs build-essential npm && \
+# Add node 15, npm and xml-js
+RUN curl -sL https://deb.nodesource.com/setup_15.x | bash -
+RUN apt-get update && apt-get install -y nodejs build-essential && \
```

Comment:
This can be consolidated into a single RUN command with `curl -sL https://deb.nodesource.com/setup_15.x | sudo -E bash -
sudo apt-get install -y nodejs` as per [https://github.com/nodesource/distributions#installation-instructions](https://github.com/nodesource/distributions#installation-instructions)